qa/tasks/cephfs/filesystem.py

```diff
@@ -389,7 +389,7 @@ def mds_signal(self, mds_id, sig, silent=False):
     def mds_is_running(self, mds_id):
         return self.mds_daemons[mds_id].running()
 
-    def newfs(self, name='cephfs', create=True):
+    def newfs(self, name='testcephfs1', create=True):
```

Comment:
Maybe you should also fix the code likes:

qa/workunits/suites/cephfs_journal_tool_smoke.sh:
```
export BIN="${BIN:-cephfs-journal-tool --rank=cephfs:0}"

And also in qa/workunits/cephtool/test.sh:

```
ceph fs new cephfs2 fs_metadata2 fs_data2
```

...
```

Reply:
Thanks! I've fixed the latter, left the former.

Reply:
What I was hoping to see here actually is a randomly generated name (this also catches any code in the QA suite which relies on the fs name rather than getting it from `self.fs`). That would make each unit test different and easier to grep in mon/client logs.

Reply:
That's a great idea. Done. Now we'll have CephFSs named as `testcephfs_abcd, where `abcd` are 4 random digits.